src/neuro_relay/__main__.py

```
@@ -233,36 +273,46 @@ async def setup_client_and_link(new_client):
             await setup_client_and_link(new_client)
         else:
-            print("[Main] Reconnection failed, relay will continue without Neuro backend")
-        
+            print(
+                "[Main] Reconnection failed, relay will continue without Neuro backend",
+            )
+
         reconnect_in_progress = False
-    
+
     async def start_outbound():
```

Ruff (0.14.8)

Missing return type annotation for private function `start_outbound`

Add return type annotation: `None`

(ANN202)